### ISSUE TITLE
[FEAT]Add keyboard navigation to the asset manager

### ISSUE THREAD

**Issue by 10leej** [Kruptein/PlanarAlly#977]
**Is your feature request related to a problem? Please describe.**
As someone currently reconfuguring my folder structute in the asset manager the current "click to select" behavious is functional, but I'm currently working on organizing the mess that is my "monster manual" folder

![image](https://user-images.githubusercontent.com/2258333/174457696-3665b7ed-1886-4e27-b583-ecf245088b4c.png)

**Describe the solution you'd like**
I'd like to be able to move selection with the keyboard (arrow keys) and be able to select multiple files by holding the shift key much like a a standard system file manager can.

**Comment by Kruptein** [comment]
Ctrl and Shift behaviour exists for mouse selection, but it should be extended to keyboard navigation yeah

**Comment by 10leej** [comment]
> Ctrl and Shift behaviour exists for mouse selection, but it should be extended to keyboard navigation yeah

I'm aware of that, but gosh my right pointer finger is worn out here.
